api/http/handlerfuncs_test.go

```diff
@@ -347,6 +347,46 @@ func TestExecGQLHandlerContentTypeJSON(t *testing.T) {
 	assert.Contains(t, users[0].Key, "bae-")
 }
 
+func TestExecGQLHandlerContentTypeJSONWithError(t *testing.T) {
+	ctx := context.Background()
+	defra := testNewInMemoryDB(t, ctx)
+	defer defra.Close(ctx)
+
+	// load schema
+	testLoadSchema(t, ctx, defra)
+
+	// add document
+	stmt := `
+{
+	"query": "mutation {
+		create_user(
+			data: \"{
+				\\\"age\\\": 31,
+				\\\"notAField\\\": true
+			}\"
+		) {_key}
+	}"
+}`
+	// remote line returns and tabulation from formatted statement
```

Comment:
typo: Do you mean replace instead of remote?

Reply:
My bad, this comment is here to indicate that the PR author's brain auto-filtered out the comments and failed to remove it having copy-pasted from the test above.

- [ ] remove?